server/migrations/1683136077244-AddUniqueConstraintToWorkspaceName.ts

```diff
@@ -0,0 +1,36 @@
+import { Organization } from 'src/entities/organization.entity';
+import { EntityManager, MigrationInterface, QueryRunner, TableUnique } from 'typeorm';
+
+export class AddUniqueConstraintToWorkspaceName1683136077244 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const entityManager = queryRunner.manager;
+    await this.migrateWorkspaceNames(entityManager);
+    await queryRunner.createUniqueConstraint(
+      'organizations',
+      new TableUnique({
+        name: 'name_organizations_unique',
+        columnNames: ['name'],
+      })
+    );
+  }
+
+  public async migrateWorkspaceNames(entityManager: EntityManager) {
+    const workspaces = await entityManager.find(Organization);
```

Comment:
we don't want to get all workspaces and iterate 
```
select sub_query.name from (select count(*) as name_count, name from organizations group by name) sub_query where name_count > 1
```
use this query to get names which repeats and iterate the array